sql/moz-fx-data-shared-prod/google_ads_derived/daily_campaign_stats_v1/query.sql

```diff
@@ -0,0 +1,5 @@
+SELECT
```

Comment:
This query needs to be added to dryrun skip list here:
https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/dryrun.py#L89

I believe this is because the CI user does not have access to the Fivetran GCP project. Once you do that dryryn CI check should succeed.